hooks/base16-sublime-merge.sh

```diff
@@ -34,8 +34,20 @@ find_replace_json_value_in_sublimemerge_settings() {
   local value=$2
   local json_file="$BASE16_SHELL_SUBLIMEMERGE_PACKAGE_PATH"/User/Preferences.sublime-settings
 
-  # Use sed to find the property and change its value
-  sed -i "s/\"$property\": \".*\"/\"$property\": \"$value\"/" $BASE16_SHELL_SUBLIMEMERGE_SETTINGS_PATH
+  # Determine the operating system
+  OS="$(uname)"
+
+  # Define the sed command based on the operating system
+  if [ "$OS" = "Darwin" ]; then
```

Comment:
I'm not sure if we have any BSD users, but I believe they'd hit this too. That being said, I don't think Sublime Merge is supported on BSD so maybe that's a non-issue.